### PR TITLE
#3935: Moving tag resolution docs to developer guide

### PR DESCRIPTION
**Pull request by abrennan89**
Partial fix for #3935 , related to #3710 

## Proposed Changes

- Move and clean up tag resolution docs
- Fixed link and other minor fixes in Operator docs

### CODE REVIEW

File: docs/admin/install/operator/configuring-serving-cr.md [hunk 1]
```diff
@@ -1,52 +1,42 @@
...
 
-## Version Configuration
+## Version configuration
 
-Cluster administrators can install a specific version of Knative Serving by using the `spec.version` field. For example,
-if you want to install Knative Serving 0.16.0, you can apply the following `KnativeServing` custom resource:
+Cluster administrators can install a specific version of Knative Serving by using the `spec.version` field.
 
-```
+For example, if you want to install Knative Serving 0.23.0, you can apply the following `KnativeServing` custom resource:
```
Comment (RichardJJG) [inline docs/admin/install/operator/configuring-serving-cr.md:19]:
```suggestion
For example, if you want to install Knative Serving v0.23.0, you can apply the following `KnativeServing` custom resource:
```

File: docs/admin/install/operator/configuring-serving-cr.md [hunk 2]
```diff
@@ -1,52 +1,42 @@
...
 spec:
-  version: 0.16.0
+  version: 0.23.0
 ```
 
-If `spec.version` is not specified, the Knative Operator will install the latest available version of Knative Serving.
-If users specify an invalid or unavailable version, the Knative Operator will do nothing. The Knative Operator always
-includes the latest 3 minor release versions. For example, if the current version of the Knative Operator is 0.16.x, the
-earliest version of Knative Serving available through the Operator is 0.14.0.
+If `spec.version` is not specified, the Knative Operator installs the latest available version of Knative Serving. If users specify an invalid or unavailable version, the Knative Operator will do nothing. The Knative Operator always includes the latest 3 minor release versions. For example, if the current version of the Knative Operator is v0.24.0, the earliest version of Knative Serving available through the Operator is v0.22.0.
```
Comment (RichardJJG) [inline docs/admin/install/operator/configuring-serving-cr.md:31]:
```suggestion
If `spec.version` is not specified, the Knative Operator installs the latest available version of Knative Serving. If users specify an invalid or unavailable version, the Knative Operator does nothing. 

The Knative Operator always includes the latest 3 minor release versions. For example, if the current version of the Knative Operator is v0.24.0, the earliest version of Knative Serving available through the Operator is v0.22.0.

Comment (abrennan89) [reply docs/admin/install/operator/configuring-serving-cr.md:31]: lol thought about doing this but wasn't sure whether to split it. Thanks.

**Review by RichardJJG — APPROVED** [review]
Thanks, @abrennan89, all looks good.

Should the tag-resolution doc have procedures instead of summaries of what the reader can do, though? Either way, I'd be okay with /lgtm at present. I suspect procedures would be better, but they could be tackled in a separate issue if this content is urgent.

**Comment by abrennan89** [comment]
> Thanks, @abrennan89, all looks good.
> 
> Should the tag-resolution doc have procedures instead of summaries of what the reader can do, though? Either way, I'd be okay with /lgtm at present. I suspect procedures would be better, but they could be tackled in a separate issue if this content is urgent.

Yeah probably, I'll leave you to open an issue for this though if you think it's worth doing since it's not really part of the scope here for just moving docs.

**Pull request merged**